docker-compose.monitoring.yml

```diff
@@ -108,6 +125,14 @@ services:
     <<: *logging
   elasticsearch:
     <<: *logging
+    container_name: elasticsearch
```

Comment:
why did you add another "elasticsearch"-services into the docker-compose.monitoring.yml ? I think this is a mistake and should be removed

Reply:
Yes, thats an error! I removed it